ssm/crypto.py

```diff
@@ -66,15 +67,15 @@ def check_cert_key(certpath, keypath):
         return False
 
     p1 = Popen(['openssl', 'x509', '-noout', '-modulus'],
-               stdin=PIPE, stdout=PIPE, stderr=PIPE)
+               stdin=PIPE, stdout=PIPE, stderr=PIPE, universal_newlines=True)
```

Comment:
Is `universal_newlines=True` added for backwards compatibility with Python 2.6? Either way, would be useful to add a comment here as to why it's needed.

Reply:
I'd say it's actually added for forward compatibility with Python 3. Python 2.7 doesn't require it. I used `universal_newlines` rather than the later `text` argument, which only exists in later versions of Pyhton 3, for backwards compatibility. Comments could be useful, though I was wary of littering the code with comments for a fairly simple change.